### PR TITLE
db update

### PR DESCRIPTION
**Pull request by schutzbot**
automated db update

### CODE REVIEW

File: .github/workflows/auto_update.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+      - name: Update DB
+        run: |
+          ./manifest-db/tools/import-image-tests --verbose composer/test/data/manifests/ manifest-db/manifest-db --manifest-only --db-ignore=manifest-db/db-ignore
+
+      - name: Send to CI
+        run: |
+          cd manifest-db
+          git config --local user.name "SchutzBot"
+          git config --local user.email "schutzbot@redhat.com"
```
Comment (gicmo) [inline .github/workflows/auto_update.yml:33]: @ochosi, what is the email we use for Schutzi?

File: .github/workflows/auto_update.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+          now=$(date '+%Y-%m-%d-%H%M%S')
+          BRANCH_NAME="db-update$now"
```
Comment (gicmo) [inline .github/workflows/auto_update.yml:36]: Maybe add a `-` between the `update` and `$now`.

File: .github/workflows/auto_update.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          git branch "$BRANCH_NAME"
+          git checkout "$BRANCH_NAME"
```
Comment (gicmo) [inline .github/workflows/auto_update.yml:38]: `checkout -b`? ;)

File: .github/workflows/auto_update.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          git add -A
+          git commit -m "db: update manifests
+
+          automatic update."
```
Comment (gicmo) [inline .github/workflows/auto_update.yml:42]: I think we can collapse that into `db: automatic update of manifests` or something.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+### Contributing
+
+Please refer to the [developer guide](https://www.osbuild.org/guides/developer-guide/developer-guide.html) to learn about our workflow, code style and more.
+
+### Repository:
+
+ - **web**:   <https://github.com/osbuild/osbuild>
+ - **https**: `https://github.com/osbuild/osbuild.git`
+ - **ssh**:   `git@github.com:osbuild/osbuild.git`
```
Comment (gicmo) [inline README.md:30]: c&p or on purpose that this points to osbuild?

**Comment by lavocatt** [comment]
This would need for PR https://github.com/osbuild/manifest-db/pull/16 to land first.

File: README.md (hunk 5 above)
Comment (lavocatt) [reply README.md:30]: I'd say, a mix of both. I think I wanted to have something pointing to osbuild, but then forgot to do it properly and left the raw section.

File: .github/workflows/auto_update.yml (hunk 1 above)
Comment (lavocatt) [reply .github/workflows/auto_update.yml:33]: I will take care of that. Thanks.

File: README.md (hunk 5 above)
Comment (lavocatt) [reply README.md:30]: I'm removing the paragraph.

**Comment by lavocatt** [comment]
closing this fake PR.

**Pull request closed without merging**